includes/admin/reports/reports.php

```diff
@@ -38,6 +38,7 @@ function give_reports_page() {
 		<h1 class="screen-reader-text"><?php echo get_admin_page_title(); ?></h1>
 
 		<h2 class="nav-tab-wrapper">
+		<a href="#" class="nav-tab">testingg</a>
```

Comment:
What's this about?

Reply:
Good catch! Yes forgot to take that out!